React-Projects/job-portal/src/components/JobDescription/JobDescription.jsx

```diff
@@ -8,16 +9,18 @@ import {
   jobDescriptionError,
   jobDescriptionContent,
 } from "../../data/JobDescriptionContent";
+import { setButtonStatus, setButtonText } from "../../store/store";
 
 const JobDescriptionCard = () => {
+  const buttonText = useSelector((state) => state.buttonText);
+  const buttonStatus = useSelector((state) => state.buttonStatus);
+  const dispatch = useDispatch();
   const jobId = useParams().id;
 
   const [jobData, setJobData] = useState({});
   const [coverLetter, setCoverLetter] = useState("");
-  const [buttonStatus, setButtonStatus] = useState(
-    "job-description-apply-button"
-  );
-  const [buttonText, setButtonText] = useState("Apply Now");
+  // const [buttonStatus, setButtonStatus] = useState(styles["apply-button"]);
+  // const [buttonText, setButtonText] = useState("Apply Now");
```

Comment:
Please remove this if not needed

Reply:
@abhi-codal I have placed this to make me remember how code of useState differs from Redux